examples/rp/.cargo/config.toml

```diff
@@ -5,4 +5,4 @@ runner = "probe-run --chip RP2040"
 target = "thumbv6m-none-eabi"        # Cortex-M0 and Cortex-M0+
 
 [env]
-DEFMT_LOG = "trace"
```

Comment:
Why change the log level?

Reply:
Trace just totally overflooded the console from the atomic-ring-buffer logs and then i thought since the examples don't depend on trace logs it might be a better default? But happy to change if you disagree :)

Reply:
True. Perhaps `debug`?

Reply:
Good idea! I'll change to that